src/main/java/com/gizasystems/automationexercise/pages/ProductQuantityPage.java

```diff
@@ -0,0 +1,65 @@
+package com.gizasystems.automationexercise.pages;
+import com.shaft.driver.SHAFT;
+import io.qameta.allure.Step;
+import org.openqa.selenium.By;
+import org.openqa.selenium.interactions.Actions;
+
+public class ProductQuantityPage {
+
+
+    // Locators
+    private final SHAFT.GUI.WebDriver driver;
+
+    private final By viewProductbtn_button = By.xpath("(//div[@class='choose']//a)[2]");
+
+    private final By verifyDetailsExist = By.xpath("//span/Label");
+
+    private final By hoverBtn_button = By.id("quantity");
+
+    private final By addToCartbtn_Button = By.xpath("//button [@class= \"btn btn-default cart\"]");
+
+    private final By viewCartbtn_button = By.xpath("//a[@href=\"/view_cart\"]/u");
+
+    private final By productAdded = By.xpath("//td[@class = \"cart_quantity\"]/button[@class=\"disabled\"]");
```

Comment:
- No need to have empty line between locators, please remove them.
- Please replace the " in your xpath to '